app/routes/screens/race/Race.js

```diff
@@ -55,8 +55,10 @@ class RacePredictor extends Component {
     }).isRequired
   };
 
-  static finish() {
-    console.log("Countdown finished");
+  static intToColor(id = 0) {
```

Comment:
one color to rule them all! 🤐 
helper function to convert int (athlete ID) to color (Map view marker color). 
So in the end, each athlete could be  easily spotted on da map! 
Nice, isn't it?